Makefile

```diff
@@ -67,6 +67,8 @@ OPERATOR_SDK_VERSION ?= 1.29.0
 
 CERTMANAGER_VERSION ?= 1.10.0
 
+ENVTEST_VERSION ?= release-0.18
```

Comment:
can we use 0.19? 


https://github.com/kubernetes-sigs/controller-runtime/releases

Reply:
![image](https://github.com/user-attachments/assets/d84ac782-fdd5-4f73-a0a0-e40fa0e01f41)
it seems can not 
https://github.com/kubernetes-sigs/controller-runtime/tree/main/tools/setup-envtest

Reply:
Could we pin this to https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.18.5 ?

Reply:
release-0.18  maybe  0.18  latest?   should we need  0.18 latest?

Reply:
If we're pinning it, I'd prefer a specific tag rather than a branch,

Reply:
ok , i got it

Reply:
go: downloading sigs.k8s.io/controller-runtime v0.18.5
go: sigs.k8s.io/controller-runtime/tools/setup-envtest@v0.18.5: module sigs.k8s.io/controller-runtime@v0.18.5 found, but does not contain package sigs.k8s.io/controller-runtime/tools/setup-envtest
it seems can not use specific tag